README.md

```diff
@@ -0,0 +1,48 @@
+# eLife Search
+
+This project is using elasticsearch to index content of the eLife articles and provide a full-text search for Journal.
+
+To reliably recreate any issue experienced in CI or Prod you should continue to use [builder](https://github.com/elifesciences/builder).
+
+## Docker for local development
+
+Important: Keep in mind that docker is just used to improve the developer experience.
+
+1. Clone the project `git clone https://github.com/elifesciences/search.git`
+2. Rename `/dev/config.php.dist` on local to `/dev/config.php` 
+3. Run `docker-compose -f dev/docker-compose.yaml up --build`
+
+### Setup
+
+Follow the steps below to set up the project:
+
+1. Go to `app` container:
+
+```bash 
+$ docker-compose -f dev/docker-compose.yaml exec app /bin/bash
+```
+
+2. Run following commands in order:
+
+```bash 
+$ bin/console queue:create
```

Comment:
Getting the following error at this step: 

```
root@e0bbc1a8fb5e:/app# bin/console queue:create

Fatal error: Uncaught Elasticsearch\Common\Exceptions\NoNodesAvailableException: No alive nodes found in your cluster in /app/vendor/elasticsearch/elasticsearch/src/Elasticsearch/ConnectionPool/StaticNoPingConnectionPool.php:64
Stack trace:
#0 /app/vendor/elasticsearch/elasticsearch/src/Elasticsearch/Transport.php(82): Elasticsearch\ConnectionPool\StaticNoPingConnectionPool->nextConnection()
#1 /app/vendor/elasticsearch/elasticsearch/src/Elasticsearch/Transport.php(99): Elasticsearch\Transport->getConnection()
#2 /app/vendor/elasticsearch/elasticsearch/src/Elasticsearch/Connections/Connection.php(297): Elasticsearch\Transport->performRequest('GET', '/key_value_stor...', Array, NULL, Array)
#3 /app/vendor/react/promise/src/FulfilledPromise.php(28): Elasticsearch\Connections\Connection->Elasticsearch\Connections\{closure}(Array)
#4 /app/vendor/ezimuel/ringphp/src/Future/CompletedFutureValue.php(55): React\Promise\FulfilledPromise->then(Object(Closure), NULL, NULL)
#5 /app/vendor/ezimuel/ringphp/src/Core.php(341): GuzzleHttp in /app/vendor/elasticsearch/elasticsearch/src/Elasticsearch/ConnectionPool/StaticNoPingConnectionPool.php on line 64
```

Reply:
@nlisgo It happens because it takes some time for elasticsearch node to be ready, if give it a minute or two, you won't see this error.